components/server/package.json

```diff
@@ -65,16 +65,16 @@
     "lodash.debounce": "^4.0.8",
     "longjohn": "^0.2.12",
     "minio": "^7.0.12",
-    "node-fetch": "^2.6.1",
+    "node-fetch": "^2.6.7",
     "opentracing": "^0.14.4",
-    "p-throttle": "^3.0.0",
+    "p-throttle": "^5.0.0",
```

Comment:
> require() of ES Module /app/node_modules/p-throttle/index.js

same issue

Reply:
`"type": "module"` in package.json?

Reply:
>>    require() of ES Module /app/node_modules/p-throttle/index.js

> same issue

What did you do? And why did the build run? :grimacing:

Reply:
server is crashing in the preview deployment

Reply:
> "type": "module" in package.json?

Not, it's not that easy. Maybe not too complicated either, but I want the other changes to be in by today. I will remove pthrottle, then.

Can you give be your cmdline to reproduce?